src/vanilla/atom.ts

```diff
@@ -40,6 +40,7 @@ type OnMount<Args extends unknown[], Result> = <
 export interface Atom<Value> {
   toString: () => string
   read: Read<Value>
+  is?: (self: Atom<unknown>, target: Atom<unknown>) => boolean
```

Comment:
Hm, it doesn't seem to add any capability.
If it works, it should work with this too:
```suggestion
  is?: (this: Atom<unknown>, target: Atom<unknown>) => boolean
```